mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -90,7 +90,7 @@ def binary_cross_entropy(pred,
     if weight is not None:
         weight = weight.float()
     loss = F.binary_cross_entropy_with_logits(
-        pred, label.float(), pos_weight=class_weight, reduction='none')
+        pred, label.float(), weight=class_weight, reduction='none')
```

Comment:
In official doc: https://pytorch.org/docs/stable/generated/torch.nn.functional.binary_cross_entropy_with_logits.html, `pos_weight` argument exists.

Reply:
But, pos_weight is for positive samples not class

Reply:
`pos_weight (Tensor, optional): a weight of positive examples. Must be a vector with length equal to the number of classes` -->class weight.
`weight (Tensor, optional): a manual rescaling weight if provided it's repeated to match the input tensor shape`  --> sample-wise weight.

Reply:
In official doc: https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html#torch.nn.BCEWithLogitsLoss
There aren't classes in BCE!
If "class" means {0 : negative samples, 1: positive samples}, that is right! I understand it!